packages/components/phone-number/src/components/osds-phone-number/osds-phone-number.tsx

```diff
@@ -175,7 +185,7 @@ export class OsdsPhoneNumber implements OdsPhoneNumberAttribute, OdsPhoneNumberE
             }}
               slot="selectedLabel"
               class="phone-number__select-label" />
-            { this.parsedCountries?.map((country) => {
+            { this.parsedCountries?.sort((a, b) => this.sortCountriesByName(a, b)).map((country) => {
```

Comment:
using the sort directly in the render will go through it on every render, would be better to sort on Watch handler to ensure we sort only once